tests/TestyMcTestFace.cpp

```diff
@@ -18,8 +20,19 @@ class FrameIsdTest : public ::testing::Test {
       json jsonIsd = json::parse(isdFile);
       isd.clearAllParams();
       for (json::iterator it = jsonIsd.begin(); it != jsonIsd.end(); ++it) {
-         isd.addParam(it.key(), it.value().dump());
+         if (it.value().size() >1 ) {
+            std::vector<double> v = it.value();
+            for (int j=0;j < v.size(); j++) {
+               std::ostringstream val;
+               val << std::setprecision(15) << v[j];
+               isd.addParam(it.key(),val.str());
```

Comment:
I would rather see precision being set in a single place - inside of the ISD.

Reply:
I think the precision can just be removed